js/deer-config.js

```diff
@@ -421,7 +424,7 @@ export default {
                         if(!filteringProps.includes("title")) li += `data-title="[ unlabeled ]"`
                         li += `>
                             ${inclusionBtn}
-                            <a target="_blank" href="${options.link}${glossID}">
+                            <a target="_blank" href="${deduplicatedList.link}${glossID}">
```

Comment:
This is causing bad links with undefined on gloss-transcription.html.  I think you want `options.link` here.
`
```HTML
<a target="_blank" href="undefinedhttps://devstore.rerum.io/v1/id/61436336e74876243131a4ea">
    <span class="serifText">Docens suos ascendere</span>
</a>
```